skipchain/skipchain.go

```diff
@@ -68,6 +69,10 @@ type Service struct {
 // added.
 func (s *Service) StoreSkipBlock(psbd *StoreSkipBlock) (*StoreSkipBlockReply, onet.ClientError) {
 	prop := psbd.NewBlock
+	if !s.ServerIdentity().Equal(prop.Roster.Get(0)) {
```

Comment:
Mhhh. Is this not hazardous ? 
Since the `prop.Roster.Get(0)` is coming from the client, the client could basically make anyone propose a block if he can re-arrange the Roster the way he wants, and it will still pass, as it's still the same roster ID or stg ?